Dockerfile

```diff
@@ -145,6 +154,8 @@ RUN apt-get update
     apt-get install -t ${UBUNTU_RELEASE} -y --no-install-recommends $RUNTIME_DEPENDENCIES                && \
     apt-get install -t ${UBUNTU_RELEASE} -y --no-install-recommends $(cat "${PREFIX_DIR}"/DEPENDENCIES)  && \
     rm -rf /var/lib/apt/lists/*
+# Reset /etc/apt/sources.list file
+RUN test -f  /etc/apt/sources.list.bak && mv /etc/apt/sources.list.bak /etc/apt/sources.list
```

Comment:
Why reset things? That would seem to add inconsistency between how the image is built and the running system??

Reply:
- I think both is ok，guacd container is not allow to apt install packages for now ,because the run user is guacd.
- After someone  install all dependencies , he can run this docker image anywhere.